### DISCUSSION TITLE
The automatic update prompt is not turned on, it will still prompt

### DISCUSSION THREAD

**Discussion by git10135405** [dbeaver/dbeaver#13015] [Q&A] [answered]
The automatic update prompt is not turned on, but when I start dbeaver, it still prompts？Is it a bug? 

![image](https://user-images.githubusercontent.com/35558611/119961305-7ca59680-bfd8-11eb-936b-72b03275011b.png)

**Comment by uslss** [comment, marked as answer]
please, uncheck higlighted option
![image](https://user-images.githubusercontent.com/31996417/120167512-801e6500-c206-11eb-9744-5f3573241229.png)

**Comment by git10135405** [comment]
![image](https://user-images.githubusercontent.com/35558611/123742589-7f005680-d8de-11eb-8cc8-118d99692a06.png)
What is the difference with this? It is easy to be mistaken for this
